app/move/app/new/controllers/health-information.ts

```diff
@@ -0,0 +1,15 @@
+import addNegativeOption from '../../../../../common/helpers/field/add-negative-option'
+// @ts-expect-error TODO: convert to TS
```

Comment:
Is this file already in typescript?

Reply:
The file it links to is still a .js file but it is a module. This comment will cause a lint error when we convert it to TypeScript, which will remind us to remove the comment.